### DISCUSSION TITLE
[v2] Requesting A example of how I access to arbitrary filesystem paths.

### DISCUSSION THREAD

**Discussion by s3xysteak** [tauri-apps/tauri#9339] [Q&A] [answered]
In `v1` I use `"scope": [ **/* ]`, but it seems that can not work in `v2`.
I get user's steam games install paths and copy mods to there. But I cannot know where user will install games, so I think I need a solution to access to arbitrary filesystem paths.

**Comment by FabianLars** [comment, marked as answer]
> But I cannot know where user will install games, so I think I need a solution to access to arbitrary filesystem paths.

You could ask them with a [folder dialog](https://beta.tauri.app/features/dialog/) which will extend the scope for the selected folder automatically and if you use the [persisted-scope](https://beta.tauri.app/features/persisted-scope/) plugin it will even persist between app starts.

Alternatively `**` should still work as part of the new ACL: https://beta.tauri.app/features/file-system/#permissions

**Reply by s3xysteak** [reply]
Thank you, it works! [Document here](https://beta.tauri.app/references/v2/js/fs/) said configure it in `plugins`. So I just configured `tauri.conf.json` and did not work.

**Comment by s3xysteak** [comment]
I hope to provide an extremely simple interactive way. Because I only need to query specific game directories, as I query the installation directory of Steam through the registry, and then parse Steam's configuration file to find the game installation directory, I hope this process is automatic and does not require manual operation by the user, so it is difficult to set the scope. 

Is there a feature that allows scope adjustment through code within a certain scope? Like:
```ts
import { createScopedAcl } from '@tauri-app/api'

const acl = createScopedAcl()
acl.permissions({
    "identifier": "fs:scope",
    "allow": [{"path": "**/*"}]
}, async () => {
  await copyFile('a', 'b')
})
```
